pkg/controller/serviceimport/serviceimport_controller.go

```diff
@@ -0,0 +1,153 @@
+package serviceimport
```

Comment:
Neither a controller, etc...

Nothing that is "Submariner" or "ServiceDiscovery" CRDs API should live in this repo.